Source/Particles/SpeciesPhysicalProperties.H

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2020 Maxence Thevenet
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef WARPX_SPECIESPHYSICALPROPERTIES_H_
+#define WARPX_SPECIESPHYSICALPROPERTIES_H_
+
+#include "Utils/WarpXConst.H"
+
+#include <AMReX_REAL.H>
+#include <AMReX_AmrCore.H>
+
+#include <map>
+#include <limits>
+
+enum struct PhysicalSpecies{unspecified=0, electron, positron, photon, hydrogen};
+
+static PhysicalSpecies from_string(std::string species)
```

Comment:
this is all very good, maybe just add a namespace such as `particles` since the parameter signature `string` is very general

Reply:
sure, done :)